### ISSUE TITLE
[Library Stabilization] Should the BigInteger number theory functions have better argument names?

### ISSUE THREAD

**Issue by lydia-duncan** [chapel-lang/chapel#17726]
`jacobi` and `kronecker` have arguments `a` and `b` and `legendre`'s arguments are `a` and `p`.  Are there more descriptive names we could use?

The GMP documentation for these functions is on this page: https://gmplib.org/manual/Number-Theoretic-Functions

**Comment by bradcray** [comment]
(You probably know this, but I didn't until I clicked through):  I'm noting that GMP uses these argument names as well, and that we've inherited them.  Of course, they arguably matter more for us since we have call-by-keyword features, and in GMP they probably only matter as a convention and for documenting the routines?

**Comment by lydia-duncan** [comment]
Yeah, we did inherit them.  GMP's argument names in general tend to be pretty succinct and not necessarily descriptive - while we could continue to follow their precedence, I think we could argue for taking a different approach ourselves

**Comment by lydia-duncan** [comment]
These functions are unstable for 2.0 because of questions about where they will live.  As a result, this issue is also not for 2.0
